conf/train/train_mlp.yml

```diff
@@ -22,5 +22,5 @@ x_dist:
 steps: 500000
 log_freq: 100
 checkpoint_freq: 5000
-model: *std_gpt2
+model: *std_llama2
```

Comment:
We should make a separate config file instead of supplanting. That'll let us verify that our code has "functional-"parity with the Garg et al. repo.

Reply:
implemented